TenancyInformationApi/V1/Boundary/Response/TenancyInformationResponse.cs

```diff
@@ -0,0 +1,19 @@
+namespace TenancyInformationApi.V1.Boundary.Response
+{
+    public class TenancyInformationResponse
+    {
+        public string TenancyReference { get; set; }
```

Comment:
Is this Tag Ref? I think the full name for Tag Ref is Tenancy Agreement Reference